projects/valtimo/components/src/lib/components/form-io/form-io.component.ts

```diff
@@ -182,8 +183,8 @@ export class FormioComponent implements OnInit, OnChanges, OnDestroy {
   private setTimerForTokenRefresh(token: string): void {
     const tokenExp = (jwt_decode(token) as any).exp * 1000;
     const expiryTime = tokenExp - Date.now() - 1000;
-    if (!this.tokenRefreshTimerSubscription) {
-      this.tokenRefreshTimerSubscription = timer(expiryTime).subscribe(() => {
+    if (!this._tokenRefreshTimerSubscription) {
+      this._tokenRefreshTimerSubscription = interval(expiryTime).subscribe(() => {
```

Comment:
The expiry time is not necessarily the interval. It's just the time left on token when initialising this component. Some other logic is needed